app/views/admin/bookstores/edit.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Edit <%=@bookstore.name %></h1>
+
+<%= render "form" %>
+<br>
+
+<h3>Reviews</h3>
+<table class="small-12 columns">
```

Comment:
I'd prefer to see some flexible `<div>`s here vs a table. Tables should typically only be used for data that needs to be displayed in a tabulated fashion, and they are difficult to render responsively for different viewport widths on various devices. Use the Foundation grid to improve the structure of this page.